lib/common/client/middleware/response/facility_parser.rb

```diff
@@ -162,6 +168,12 @@ def dental_services(facility_id)
 
               services
             end
+
+            def get_mapper(key)
+              @mapping['mappers']&.find do |mapping_key, mapper|
+                break mapper if mapping_key.eql?(key)
```

Comment:
no need to break when using `.find`. Just this should work 
```suggestion
                mapping_key.eql?(key)
```

Reply:
> no need to break when using `.find`. Just this should work

Without the break, I believe it returns an array so it would return [{'classification' => my_method }]